clients/privacy-center/pages/consent.tsx

```diff
@@ -214,16 +221,61 @@ const Consent: NextPage = () => {
     redirectToIndex,
   ]);
 
+  /**
+   * Initialize internalization library.
+   */
+  const experience = useAppSelector(selectPrivacyExperience);
+  const { IS_OVERLAY_ENABLED } = settings;
+  const isConfigDrivenConsent = !IS_OVERLAY_ENABLED;
+  const [isI18nInitialized, setIsI18nInitialized] = useState(false);
+
+  useEffect(() => {
+    const i18n = setupI18n();
+
+    // 1. Config driven consent
+    if (isConfigDrivenConsent) {
+      // If we're in config driven consent then
+      // still load the static messages in order to have default english
+      // messages available
+      loadMessagesFromFiles(i18n);
+      setI18nInstance(i18n);
+      setIsI18nInitialized(true);
+      return;
+    }
+
+    // 2. Notice driven consent
+    if (!experience) {
+      return;
+    }
+
+    initializeI18n(
+      i18n,
+      window?.navigator,
+      experience as PrivacyExperience,
+      {
+        debug: true,
+      },
```

Comment:
should this be hardcoded here?

Reply:
I'll change it to check for the node environment to be development.